features/step_definitions/user_steps.rb

```diff
@@ -267,4 +268,11 @@ def get_series_name(age, classname, name)
 Then /^I should get confirmation that I changed my username$/ do
   step(%{I should see "Your user name has been successfully updated."})
 end
- 
\ No newline at end of file
+ 
+When(/^I am login in to the current page using "([^"]*)" as the username and  "([^"]*)" as the password$/) do |username, password|
```

Comment:
In fact, I would even argue that there is no need for an additional step, since this is almost identical to `I am logged in as "([^\"]*)" with password "([^\"]*)"` on line 39, which could be extended to have a `(to the current page)` clause that turns off the `visit login_url` line. For example: